tasks/bootstrap_local_vm/02_create_local_vm.yml

```diff
@@ -1,14 +1,18 @@
 ---
 - name: Create hosted engine local vm
   block:
+  - set_fact: host_key_checking="{{ lookup('config','HOST_KEY_CHECKING') or true }}"
```

Comment:
something OR true will always return true.
I think you can safely remove the 'or true' since HOST_KEY_CHECKING is a valid config key.

Reply:
Done